### PR TITLE
docs(log): document public interfaces

### PR DESCRIPTION
**Pull request by kt3k**
This PR fixes the errors from the command `deno doc --lint log/mod.ts`

### CODE REVIEW

File: log/base_handler.ts [hunk 1]
```diff
@@ -8,19 +8,25 @@ import {
...
 }
 
+/** The base class of handlers */
 export abstract class BaseHandler {
   #levelName: LevelName;
   #level: LogLevel;
+  /** The log formatters */
   formatter: FormatterFunction;
 
+  /** Construct a {@linkcode BaseHandler} */
```
Comment (iuioiua) [inline log/base_handler.ts:29]: Ditto for the other constructors
```suggestion
  /** Constructs a new instance. */
```

**Review by iuioiua — CHANGES_REQUESTED** [review]
Maybe we should clarify the title, as this PR only adds descriptions to public APIs.

**Comment by iuioiua** [comment]
Also, I feel like these descriptions aren't particularly informative 😬

**Comment by kt3k** [comment]
> Also, I feel like these descriptions aren't particularly informative 😬

At least this reduces lots of lint errors

**Pull request closed without merging**